package.json

```diff
@@ -4,31 +4,35 @@
   "private": true,
   "dependencies": {
     "@adobe/aio-lib-core-logging": "^2.0.0",
+    "@adobe/aio-lib-files": "^3.0.1",
+    "@adobe/aio-lib-state": "^2.1.2",
```

Comment:
Are we using lib-state? It will be good to add only those libs that will be used and update later as necessary.